.github/workflows/cypress-component-parallelization.yml

```diff
@@ -58,6 +58,69 @@ jobs:
           path: centreon/cypress/results/*.json
           retention-days: 1
 
+      - name: Change coverage report name
+        run: mv ${{ inputs.module_name }}/.nyc_output/out.json ${{ inputs.module_name }}/.nyc_output/${{ matrix.spec }}-out.json
+        shell: bash
+
+      - name: Archive test coverage
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ inputs.name }}-coverage
+          path: ${{ inputs.module_name }}/.nyc_output/${{ matrix.spec }}-out.json
+          retention-days: 1
+
+  cypress-component-coverage-report:
+    needs: [cypress-component-test-run]
+    if: always()
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - name: Download coverage reports
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.name }}-coverage
+          path: .nyc_output/
+
+      - name: Merge coverage reports
+        run: |
+          pnpx nyc merge .nyc_output
+          mv coverage.json .nyc_output/out.json
+        shell: bash
+
+      - name: Generate lcov coverage report
+        run: pnpx nyc report --reporter=lcov --reporter html
+        shell: bash
+
+      - name: Format title
+        uses: frabert/replace-string-action@v2
+        id: title
+        with:
+          pattern: '/'
+          string: ${{ inputs.module_name }}
+          replace-with: '-'
+          flags: 'g'
```

Comment:
use `sed` instead of using an external action : 
```
MODULE_NAME=`echo "${{ inputs.module_name }}" | sed -e "s#/#-#g"`
```
then export MODULE_NAME as env variable or github output